checksec

```diff
@@ -86,9 +86,7 @@ FS_cnt_checked=0
 FS_cnt_unchecked=0
 FS_libc=0
 
-if [[ $(id -u) != 0 ]]; then
-  export PATH=${PATH}:/sbin/:/usr/sbin/
-fi
+export PATH=${PATH}:/sbin/:/usr/sbin/
```

Comment:
This needs added to the source files so that it gets generated (in the src directory)